explainer.md

```diff
@@ -28,7 +28,7 @@ To balance this tension we propose an API with the following key properties:
 - By separating the act of requesting from the specific protocol, we can enable flexibility and adaptability in both the protocol and credential formats. This way, the pace of changes in browsers won't hinder progress or block new developments.
 - Require request transparency, enabling user-agent inspection for risk analysis
 - Assume response opacity (encrypted responses), enabling verifiers and holders to control where potentially sensitive PII is exposed
-- Prevent website from silently querying for the availability of digital credentials and communicating with wallet providers without explicit user consent 
+- Sites learn nothing without user permission. Prevent websites from silently querying for the availability of digital credentials and communicating with wallet providers prior to the user confirming each action.
```

Comment:
I think "permission" is just one aspect (but it might not be the right word because it's not really a permission prompt... not in the traditional sense, like with Geolocation)... 

How about: 
```suggestion
- Websites cannot access any information without the user's interaction. This API ensures that sites cannot silently query for digital credentials or communicate with wallet providers without the user's active participation and confirmation of each action.
```